README.md

```diff
@@ -12,7 +12,7 @@ A Client library for OAuth2 which uses JWT for the generation of access tokens.
               .subject("myapp ")
               .build();
 
-  TokenSource tokenSource = config.tokenSource();
+  TokenSource tokenSource = TokenSources.reusableTokenSource(new OkHttpTokenSource(config));
```

Comment:
> Also this is jwt-java-client - I would not use this client library if I have to make requests to the a services that do not use jwt token authorization.

It's just a helper classes which are easy to be hooked in the existing HTTP client which is used in the project. Both okhttp and google-http-client are providing good API for hooking interceptors. 
Actually the builders should build the interceptors, not the clients as they are overlay. I'll think for simplification of this API. 

> we've got JwtConfig, TokenSources, TokenSource, OkHttpTokenSource, BearerAuthenticationInterceptor, only for a setup. can't we just have new OkHttpClient.Builder(new BearerAuthenticationInterceptor(json)).build() or
> new OkHttpClient.Builder(new ServiceAccountCredentials(json)).build()
where we can pass and the credentials separated by properties values new OkHttpClient.Builder(new ServiceAccountCredentials(clinetEmail, privatekey)).jwtKey("jwt key").build()